omero-web/omero-install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Setup and run omero
+
+set -eux
+
+git clone -b java-parameters https://github.com/aleksandra-tarkowska/omero-install.git /tmp/omero-install
```

Comment:
This should be replaced once https://github.com/jburel/omero-install/pull/2 is merged with https://github.com/ome/omero-install/pull/94 and all merged into main line